### PR TITLE
Add a note about exposing env vars for unrecognized Docker services

### PR DESCRIPTION
**Pull request by fabpot**
This one should be merged when Symfony CLI 4.21 is released.

### CODE REVIEW

File: setup/symfony_server.rst [hunk 1]
```diff
@@ -299,6 +299,9 @@ project. It understands that this is a MySQL service and creates environment
 variables accordingly with the service name (``database``) as a prefix:
 ``DATABASE_URL``, ``DATABASE_HOST``, ...
 
+If the service is not in the supported list below, generic environment
+variables are set: ``HOST``, ``IP``, and ``HOST``.
```
Comment (wouterj) [inline setup/symfony_server.rst]: This list contains duplicate "HOST"

Comment (tucksaun) [inline setup/symfony_server.rst]:
```suggestion
variables are set: ``HOST``, ``IP``, and ``PORT``.
```

File: setup/symfony_server.rst [hunk 2]
```diff
@@ -361,6 +364,20 @@ When Docker services are running, browse a page of your Symfony application and
 check the "Symfony Server" section in the web debug toolbar; you'll see that
 "Docker Compose" is "Up".
 
+.. note::
+
+    If you don't want environment variables to be exported for a service, set
```
Comment (tucksaun) [inline setup/symfony_server.rst]:
```suggestion
    If you don't want environment variables to be exposed for a service, set
```
?

**Comment by fabpot** [comment]
All fixed

**Comment by javiereguiluz** [comment]
Symfony CLI 4.21 was released yesterday, so let's merge! Thanks Fabien!

**Pull request merged**